### DISCUSSION TITLE
VPN entites

### DISCUSSION THREAD

**Discussion by webchl** [dbarzin/mercator#165] [Ideas]
Bonjour Didier,
 
J’aurais une question concernant les connexions VPN (SSL , IPSEC…) avec nos différents prestataires, à savoir comment et à quel endroit faut il les référencer ?
Il serait peut etre bien dans de futures versions, un lien entre les entités, VPN, serveurs/applications
Qu’en penses tu ?
 
Merci

**Comment by dbarzin** [comment]
Les connexions distantes peuvent être inventoriées au travers de la liste des entités extérieures connectées dans la vue des infrastructures logiques. La table n'est sans doutes pas complète mais nous pouvons discuer des attributs à ajouter.
